src/components/header.js

```diff
@@ -6,6 +6,7 @@ const Header = ({children, style}) => {
 		<AppBar 
 			style={style}
 			title={children}
+			titleStyle={ { position: "absolute", right: "10px" } }
```

Comment:
Seems like the material-ui convention is to put the text [title on the left and the icon on the right](http://www.material-ui.com/#/components/app-bar), so I swapped them.

![image](https://cloud.githubusercontent.com/assets/357481/13659416/4e2474c4-e64f-11e5-9042-7253bb1b47bc.png)